wherobots/db/cursor.py

```diff
@@ -75,10 +75,16 @@ def fetchmany(self, size: int = None):
         return results
 
     def fetchall(self):
-        return self.__get_results()[self.__current_row :]
+        return self.__get_results()[self.__current_row:]
+
+    def close(self):
+        """Close the cursor."""
```

Comment:
I think it's better/best practice to add a `pass` statement for empty functions in Python.